external-crates/move/crates/move-analyzer/editors/code/README.md

```diff
@@ -21,6 +21,13 @@ definition, type on hover). Please see [Troubleshooting](#troubleshooting) for s
 the pre-built move-analyzer binary is not available for your platform or if you want to use move-analyzer
 binary stored in a different location.
 
+If you want to build and test Move code using the extension, you must install the `sui` binary on
+your machine - see [here](https://docs.sui.io/guides/developer/getting-started/sui-install) for
+instructions. The extension assumes that the `sui` binary is in your system path, but you can set
+its custom location location using VSCode's settings (`⌘,` on macOS, or use the menu item *Code >
```

Comment:
Should it be `⌘` + `,`?
Not sure the style guide on such things

Reply:
I just copied what was already out there in the old version :-) but this style is more readable so let's adopt it!